src/consts.ts

```diff
@@ -0,0 +1,5 @@
+export const WINDOW_EXISTS = typeof window !== "undefined";
```

Comment:
What if we made the check the opposite and named it something like `SERVER_SIDE_RENDERING`?

Reply:
But how do you know that when window is undefined it is rendered on server side? We may also use similar environment to React Native, which doesn't run on server.

Reply:
I think for RN you need way more logic (like AsyncStorage). For that to work, we would have to split the SDK and initialize it like you initialize Apollo:

```typescript
import { createStorage } from "@saleor/browser";
import { SaleorManager } from "@saleor/sdk";

const storage = createStorage();
const client = new SaleorManager({ storage });
```

(Ignore the names, it's all just for illustrative purposes.)